pkg/controllers/management/auth/user.go

```diff
@@ -4,31 +4,38 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/rancher/types/apis/core/v1"
 	"github.com/rancher/types/apis/management.cattle.io/v3"
 	"github.com/rancher/types/config"
+	v12 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/cache"
 )
 
 type userLifecycle struct {
-	prtb         v3.ProjectRoleTemplateBindingInterface
-	crtb         v3.ClusterRoleTemplateBindingInterface
-	grb          v3.GlobalRoleBindingInterface
-	users        v3.UserInterface
-	tokens       v3.TokenInterface
-	prtbLister   v3.ProjectRoleTemplateBindingLister
-	crtbLister   v3.ClusterRoleTemplateBindingLister
-	grbLister    v3.GlobalRoleBindingLister
-	prtbIndexer  cache.Indexer
-	crtbIndexer  cache.Indexer
-	grbIndexer   cache.Indexer
-	tokenIndexer cache.Indexer
+	prtb             v3.ProjectRoleTemplateBindingInterface
+	crtb             v3.ClusterRoleTemplateBindingInterface
+	grb              v3.GlobalRoleBindingInterface
+	users            v3.UserInterface
+	tokens           v3.TokenInterface
+	namespaces       v1.NamespaceInterface
+	prtbLister       v3.ProjectRoleTemplateBindingLister
+	crtbLister       v3.ClusterRoleTemplateBindingLister
+	grbLister        v3.GlobalRoleBindingLister
+	prtbIndexer      cache.Indexer
+	crtbIndexer      cache.Indexer
+	grbIndexer       cache.Indexer
+	tokenIndexer     cache.Indexer
+	namespaceIndexer cache.Indexer
```

Comment:
You dont need an indexer, since you just need to get by name. you can just use the lister. `lister.Get()`

Reply:
Duh, thank you!